source/js/components/join/join.jsx

```diff
@@ -200,19 +201,11 @@ export default class JoinUs extends React.Component {
       label: `Signup form input focused`
     });
 
-    let emailFields = document.querySelectorAll(
```

Comment:
Interaction in one form shouldn't introduce UI changing for ALL signup form instances & we should avoid manipulating DOM elements directly using `document.blah` in React.